Lib/sysconfig.py

```diff
@@ -95,20 +95,20 @@ def joinuser(*args):
     _INSTALL_SCHEMES |= {
         # NOTE: When modifying "purelib" scheme, update site._get_path() too.
         'nt_user': {
-            'stdlib': '{userbase}/lib/python{py_version_short}',
-            'platstdlib': '{userbase}/lib/python{py_version_short}',
-            'purelib': '{userbase}/lib/python{py_version_short}/site-packages',
-            'platlib': '{userbase}/lib/python{py_version_short}/site-packages',
-            'include': '{userbase}/include/python{py_version_short}',
+            'stdlib': '{userbase}/lib/python{py_version_plat}',
```

Comment:
What's `py_version_plat`? Why not use `py_version_nodot_plat` like upstream?

Reply:
I thought it will break how it was done previously? ie. `{userbase}/lib/python3.9` vs `{userbase}/lib/python39`. If that change is fine; we can use that.

Reply:
yes, but when upgrading from 3.9 to 3.10 these paths change anyway, so no (new) breakage. Or am I missing something?

Reply:
ah, looking at our old patches I understand why you used this. I need to look through things some more...

Reply:
ok, I think I can follow now. Maybe we can call it `py_version_short_plat` instead? So it's clear it's an extension of `py_version_short`

Reply:
> Maybe we can call it py_version_short_plat instead? So it's clear it's an extension of py_version_short

Fine.

Reply:
Done